examples/cartesian_path.py

```diff
@@ -76,16 +81,36 @@
 visualize_frames(viz, "cartesian_plan", tforms_to_show, line_length=0.05, line_width=1)
 time.sleep(0.5)
 
+curr_time = 0
+
 plt.ion()
 plt.figure()
-plt.title("Joint Position Trajectories")
+plt.title("Pyroboplan - Joint Position Trajectories")
 for idx in range(q_vec.shape[0]):
     plt.plot(t_vec, q_vec[idx, :])
-
-curr_time = 0
 time_line = plt.axvline(x=curr_time, color="k", linestyle="--")
+plt.legend(model.names[1:])
 
+# TOPPRA
+vlims = np.ones(q_start.shape[0]) * max_linear_velocity
+alims = np.ones(q_start.shape[0]) * max_linear_acceleration
+# Define the geometric path and two constraints.
+path = ta.SplineInterpolator(t_vec, q_vec.T)
+pc_vel = constraint.JointVelocityConstraint(vlims)
+pc_acc = constraint.JointAccelerationConstraint(alims)
+
+instance = algo.TOPPRA([pc_vel, pc_acc], path, parametrizer="ParametrizeConstAccel")
+jnt_traj = instance.compute_trajectory()
+
+ts_sample = np.linspace(0, jnt_traj.duration, 100)
+qs_sample = jnt_traj(ts_sample)
```

Comment:
One more thing -- the animation after you press "Enter" still uses the joint trajectory before time parameterization with TOPP-RA. You probably want to change it to use this new retimed version!

Reply:
Ok, I will change that.